src/nls/messages_zh-Hans.json

```diff
@@ -1,256 +1,256 @@
 {
   "zh-Hans": {
-    "carbon.listBoxMenuIcon.close.menu": "",
-    "carbon.listBoxMenuIcon.open.menu": "",
-    "carbon.listBoxSelection.clear.all": "",
-    "carbon.listBoxSelection.clear.selection": "",
-    "carbon.table.all.collapse": "",
-    "carbon.table.all.expand": "",
+    "carbon.listBoxMenuIcon.close.menu": "关闭菜单",
+    "carbon.listBoxMenuIcon.open.menu": "开启菜单",
+    "carbon.listBoxSelection.clear.all": "清除所有选择的项目",
+    "carbon.listBoxSelection.clear.selection": "清除选择的项目",
+    "carbon.table.all.collapse": "折叠所有行",
+    "carbon.table.all.expand": "展开所有行",
     "carbon.table.all.select": "选择所有行",
     "carbon.table.all.unselect": "取消选择所有行",
     "carbon.table.batch.cancel": "取消",
     "carbon.table.batch.item.selected": "选中 {totalSelected, plural, =0 {0 items} one {1 item}}",
     "carbon.table.batch.items.selected": "选中 {totalSelected, plural, other {# items}}",
```

Comment:
When selecting a single item on a table using the checkboxes the string is not displayed correctly:

![image](https://user-images.githubusercontent.com/2829095/106191708-21d36e00-61a3-11eb-996b-06901b9f8781.png)

This is due to a limitation of the component library we're using. It requires separate strings for 0/1 and plural. To work around this for Chinese we can use the same value for both strings:

```diff
-"carbon.table.batch.item.selected": "选中 {totalSelected, plural, =0 {0 items} one {1 item}}",
+"carbon.table.batch.item.selected": "选中 {totalSelected, plural, other {# items}}",
"carbon.table.batch.items.selected": "选中 {totalSelected, plural, other {# items}}",
```

The word 'item' should also be translated here or removed.